augur/tasks/git/facade_tasks.py

```diff
@@ -125,16 +125,9 @@ def update_analysis_log(repos_id,status):
 
         # If there's a commit still there, the previous run was interrupted and
         # the commit data may be incomplete. It should be trimmed, just in case.
-        for commit in working_commits:
```

Comment:
What is happening here appears to be simply a refactoring of how working commits are trimmed. Basically putting it into a method. Is that right, @IsaacMilarky ?

Reply:
Yes exactly this also uses a bulk operation instead of sending a query for every commit to trim.